__tests__/app.test.js

```diff
@@ -89,3 +89,45 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("GET 200 should get all aricles with the properties in the data file except the body property", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+        });
+      });
+  });
+  test("GET 200 should have a comment count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        articles.forEach((article) => {
+          expect(typeof article.comment_count).toBe("number");
+          if (article.article_id === 3) {
+            expect(article.comment_count).toBe(2);
```

Comment:
You could condense this test and instead,  add in ` expect(typeof article.comment_count).toBe("string");` in the test above 💡